rallyfire.py

```diff
@@ -21,30 +21,48 @@ def main(args):
     options = [opt for opt in args if opt.startswith('--')]
     args    = [arg for arg in args if arg not in options]
     server, user, password, apikey, workspace, project = rallyWorkset(options)
-    #print(" ".join(["|%s|" % item for item in [server, user, password, apikey[:8], workspace, project]]))
+    #print(f" |{server}| |{user}| |{password}| |{apikey[:8]}| |{workspace}| |{project}|")
 
     # If you want to use BasicAuth, use the following form
-    #rally = Rally(server, user, password, workspace=workspace, project=project) 
+    rally = Rally(server, user, password, workspace=workspace, project=project) 
 
     # If you want to use API Key, you can use the following form
+    #rally = Rally(server, apikey=apikey, workspace=workspace, project=project,
+    #              isolated_workspace=True)
     #rally = Rally(server, apikey=apikey, workspace=workspace, project=project)
 
     # the following form of obtaining a Rally instance will use the apikey if it is present (non None)
     # otherwise it will use the user and password for BasicAuth
     # add in the debug=True keyword arg pair if you want more verbiage ...
-    rally = Rally(server, user, password, apikey=apikey, 
-                  workspace=workspace, project=project,
-                  debug=True, isolated_workspace=True)
-    specified_workspace = workspace
+    #apikey = '_lsMzURZTRyBoD3bwnpn5kUZvDQkRIoEeGkq7QNkg'
+    #apikey = '_DFyvCutVTKAxibNJ7mHvABywT3UIwtspVjJNWf40'
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Medium

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- 37e1ee6c-5077-40d1-92e5-b479946b60ab -->